dxp/react_rendering.md

```diff
@@ -0,0 +1,148 @@
+# React Rendering in DXP
+
+This article describes DXP infrastructure to render React components from the server side perspective. This includes taglibs as well as low-level services.
+
+## React Renderer Service
+
+-   OSGi bundle: `com.liferay.portal.template.react.renderer.api`
+-   Interface name: `com.liferay.portal.template.react.renderer.ReactRenderer`
+-   Service type: `singleton`
```

Comment:
Could we provide links to source code for some of these things?
- OSGi bundle
- ReactRenderer Interface
- ...